drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -91,6 +93,7 @@ class Hmis::Form::Definition < ::GrdaWarehouseBase
     :REFERRAL,
     :REFERRAL_REQUEST,
     :EXTERNAL_FORM,
+    :CE_REFERRAL_STEP,
```

Comment:
To discuss - do we want this under DATA_COLLECTION_FEATURE_ROLES. Do we want referral functionality to be "toggled on and off" using form instances?

I think CE Referral may be too complicated for this data collection feature concept. Maybe we could leave it out for now, until we have more of an intentional conversation about how Coordinated Entry is configured.

Reply:
Remove from Data Collection Feature roles, and add as "misc" role next to occurrence point form.

Add issue under https://github.com/open-path/Green-River/issues/7190 for CE configuration, questions like:
- turn on ce for project
- choose which templates apply to the project
- does it accept and/or send out direct referrals
- prioritized client lists

Reply:
Thank you! Linking https://github.com/open-path/Green-River/issues/7409 which is I think what you meant!